tests/utils/verification/volumeproperties.go

```diff
@@ -130,15 +133,15 @@ func getVolumeStatusHost(name, hostName string) string {
 	return out
 }
 
-// VerifyDetachedStatus - check if the status gets detached within the timeout
-func VerifyDetachedStatus(name, hostName, esxName string) bool {
-	log.Printf("Confirming detached status for volume [%s]\n", name)
+// VerifyDetachedStatus- check if the status gets detached within the timeout
```

Comment:
minor: golint is complaining about this method document .. 

Reference: https://ci.vmware.run/vmware/docker-volume-vsphere/763

`../tests/utils/verification/volumeproperties.go:136:1: comment on exported function VerifyDetachedStatus should be of the form "VerifyDetachedStatus ..."`